src/main/java/io/streamzi/strombrau/router/kafka/KafkaInputConsumer.java

```diff
@@ -42,7 +42,7 @@ public void start() throws Exception {
 
             stream.subscribe(data -> {
 
-                logger.fine("pumping data from Kafka to EB");
+                logger.severe("pumping data from Kafka to EB");
```

Comment:
Is severe correct?

Reply:
it's more for debugging for now - had network issues ;-)